app/assets/stylesheets/govuk_publishing_components/components/_previous-and-next-navigation.scss

```diff
@@ -60,9 +60,9 @@
   display: inline-block;
   margin-top: .1em;
   text-decoration: underline;
-  margin-left: $gutter;
+  margin-left: govuk-spacing(5);
 
-  @include media(mobile) {
-    margin-left: $gutter-half + 9;
+  @include govuk-media-query($from: tablet) {
+    margin-left: govuk-spacing(6);
```

Comment:
Gutter half was 15px, plus 9 is 24px, so would `govuk-spacing(5)` be more appropriate to replace it?

Reply:
Yep, `govuk-spacing(5)` it is, on line L63. The CSS is now written mobile-first, starting with `govuk-spacing(5)` on mobile then going to `govuk-spacing(6)` on tablet and desktop.